theme/static/js/hs-vue/notifications-app.js

```diff
@@ -260,6 +270,11 @@ $(document).ready(function () {
                             vue.deliverTask(task);
                         }
                         break;
+                    case "resource copy to user zone":
+                        if (task.status === "completed" && task.payload) {
```

Comment:
I don't think task.payload needs to be check here and on the other tasks that don't use it.

Reply:
Yeah, agreed. Currently payload is being used in the notification for page refresh, and if I don't return a payload in the task, the notification link does not work, I tried to make it work without a payload, but could not succeed, so just took an easy path to return the resource id as the payload for this task even though the payload is not really needed for this specific task. I'll take another stab to see if I can get notification refresh in vue js to work without a payload.